### PR TITLE
build: release 2.0.0

### PR DESCRIPTION
**Pull request by typfel**
# What's new in this PR

Build release 2.0.0

----
##### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by coriolinus — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,86 @@
 # Changelog
 
+## v2.0.0 - 2024-12-02
+
+### Breaking changes
+
+- `CoreCryptoLogger` takes an additional `context` parameter.
+- `CoreCryptoError` and it child errors have been refactored to reduce the amount of error we expose and provide explicit
```
Comment (istankovic) [inline CHANGELOG.md:8]: Typo: `it` -> `its`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,86 @@
 # Changelog
 
+## v2.0.0 - 2024-12-02
+
+### Breaking changes
+
+- `CoreCryptoLogger` takes an additional `context` parameter.
+- `CoreCryptoError` and it child errors have been refactored to reduce the amount of error we expose and provide explicit 
+   errors for proteus errors. The errors we have removed will appear under the `Other` case.
```
Comment (istankovic) [inline CHANGELOG.md:9]: Shouldn't we write `proteus` with an uppercase `p`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,86 @@
...
+       WrongEpoch,
+       MessageEpochTooOld,
+       SelfCommitIgnored,
+       UnmergedPendingGroup,
+       StaleProposal,
+       StaleCommit,
+       Other(String)
+   }
+   ```
+
```
Comment (istankovic) [inline CHANGELOG.md:31]: We're missing
```
- dropped support for `i686-linux-android` target
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,86 @@
...
+### Highlights
+
+- The number of public errors have been reduced and simplified. It's no longer necessary to use the
```
Comment (istankovic) [inline CHANGELOG.md:35]: Typo: `have been` -> `has been`.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,87 @@
 # Changelog
 
+## v2.0.0 - 2024-12-02
+
+### Breaking changes
```
Comment (istankovic) [inline CHANGELOG.md:5]: I would consider moving this section just below the `Highlights` one, so the highlights come first.

**Review by istankovic — APPROVED** [review]

**Pull request merged**